bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/forms2/contract/form/FormType.java

```diff
@@ -0,0 +1,16 @@
+package org.bahmni.module.bahmnicore.forms2.contract.form;
+
+public enum FormType {
+
+    ALL_OBSERVATION_TEMPLATE_FORMS("v1"), FORM_BUILDER_FORMS("v2");
```

Comment:
Also, do we need to name them as "All_OBSERVATION_TEMPLATE_FORMS", "FORM_BUILDER_FORMS"? We shouldn't mix them up with the "means" of how they are rendered/created. They represent two different form techniques. IMO, best would be describe them so. As of now, all are obs forms

Reply:
This ENUM is introduced for better code readability only. Even though they all are obs irrespective of the technique, the way we fetch them from DB is different so it is better to have appropriate technique names instead of v1 or v2 for better code readability.

Reply:
imo: you are interpreting "v1", "v2" from the request parameters. If readability is of concern, then the API should be used right? how would this line read then
> FormType. FORM_BUILDER_FORMS.get().equals(formType)

Reply:
Correct, the intention was to represent v1 and v2 as some readable names in code level.